usr/lib/ep11_stdll/ep11_specific.c

```diff
@@ -9585,3 +9585,110 @@ static void free_ep11_target_for_apqn(target_t target)
     }
 }
 
+CK_RV token_specific_set_attribute_values(STDLL_TokData_t *tokdata, OBJECT *obj,
+                                          TEMPLATE *new_tmpl)
+{
+    ep11_private_data_t *ep11_data = tokdata->private_data;
+    CK_OBJECT_CLASS class;
+    size_t keyblobsize = 0;
+    CK_BYTE *keyblob;
+    DL_NODE *node;
+    CK_ATTRIBUTE *ibm_opaque_attr = NULL;
+    CK_ATTRIBUTE_PTR attributes = NULL;
```

Comment:
Just for better reading: I would either always use CK_ATTRIBUTE *attr or CK_ATTRIBUTE_PTR attr. I personally prefer the standard way, coding ptr variables with the '*'.

Reply:
I tend to use CK_ATTRIBUTE_PTR if it is an attribute array, and CK_ATTRIBUTE * if it is a single attribute. Not that this really makes a difference though....